<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"><head>

<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<!-- 通用css -->

<link href="/publish/zhpyen/css/common.css" rel="stylesheet" type="text/css" />
<link href="/publish/zhpyen/css/grid.css" rel="stylesheet" type="text/css" />
<!-- IE8 兼容 -->
<meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7" />
<link rel="shortcut icon" type="image/x-icon" href="/public/icon/favicon.ico" />
<!-- 引入jquery1.4 -->
<script src="/public/jquery/jquery-1.4.min.js" type="text/javascript"></script>
<script language="javascript" type="text/javascript">
/*
*截取信息简介
*summaryStr:String 信息简介
*lengTh:int 截取的长度
*/
 function cutSummary(summaryStr,lengTh){
   if(summaryStr.length>lengTh){
     summaryStr=summaryStr.substring(1,lengTh)+"...";
   }
   document.write(summaryStr);
 }
/*
*jQuery:所有外域链接在新窗口打开
*/
$(document).ready(function() {
 $("a[href*='http://']:not([href*='"+location.hostname+"']),[href*='https://']:not([href*='"+location.hostname+"'])")
 .addClass("external")
 .attr("target","_blank")
});
</script>

      
 
   

 

  <title>
Zhou Pei-Yuan Center for Applied Mathematics
</title><link href="/publish/zhpyen/css/style_second.css" rel="stylesheet" type="text/css" />
</head>
<body>
<SCRIPT language="JavaScript" src="/public/plugin/source/global.js"></SCRIPT>
<!-- 导航下拉菜单 -->
<link type="text/css" rel="stylesheet" href="/public/plugin/source/nav/css/mt_dropdown.css" />

<SCRIPT language="JavaScript" src="/public/plugin/source/nav/js/mt_dropdownc.min.js"></SCRIPT>

<SCRIPT language="JavaScript" src="/public/plugin/source/nav/js/mt_dropdown_initialize.min.js"></SCRIPT>

<script language="JavaScript">

<!--

function MM_openBrWindow(theURL,winName,features) { //v2.0

  window.open(theURL,winName,features);

}
$(document).ready(function(){

  init();

});

//-->
</script>


<!-- 导航下拉菜单 --><SCRIPT language="JavaScript" src="/public/plugin/source/global_twain.js"></SCRIPT>
 <!--container开始处-->
 <div  id="container">
  <!--header开始处-->
 
<div id="header">
   <!--logo开始处-->
   <div id="logo">
    <a href="http://www.tsinghua.edu.cn" title="清华大学"></a>
   </div>
   <!--logo结束处-->
   <!--header_mid开始处-->
 <div id="header_mid">
    <h1>周培源数学研究中心</h1>
    <p>Zhou Pei-Yuan Center for Applied Mathematics</p>
   </div>
   <!--header_mid结束处-->
   <!--header_r开始处-->
   <div id="header_r">
    <ul>
     <li><a href="/publish/zhpy/index.html" title="周培源数学研究中心"><img src="/publish/zhpyen/images/cn_text.gif" style="width:50px;height:20px"></a></li>
     <li><a href="/publish/th/index.html" title="清华大学中文主页"><img src="/publish/zhpyen/images/thu_text.gif" style="width:50px;height:20px"></a></li>
     <li>search</li>
     <li class="set_border">
     <form id="searchForm" action="#" method="post" target="_blank">
<input type="hidden" id="searchVpath" name="vpath" value=""/>
     <input type="text" name="Q"  class="txt" tabindex="1" />
     </form>
</li>
    </ul>
   </div>
   <!--header_r结束处-->
  </div>  <!--header结束处-->
  <!--nav开始处-->

<SCRIPT language="JavaScript">
function initMenuData(){
if (mtDropDown.isSupported()) {
var ms = new mtDropDownSet(mtDropDown.direction.down, 0, 0, mtDropDown.reference.bottomLeft);


var menu1170 = ms.addMenu(document.getElementById("menu_1170"));
menu1170.addItem("Areas","/publish/zhpyen/1181/index.html","");
        menu1170.addItem("Lecture","/publish/zhpyen/1182/index.html","");
        menu1170.addItem("Seminar","/publish/zhpyen/5764/index.html","");
        var menu1168 = ms.addMenu(document.getElementById("menu_1168"));
menu1168.addItem("Faculty","/publish/zhpyen/1176/index.html","");
        menu1168.addItem("Staff","/publish/zhpyen/1177/index.html","");
        menu1168.addItem("Postdoc","/publish/zhpyen/1178/index.html","");
        menu1168.addItem("PhD","/publish/zhpyen/5763/index.html","");
        var menu1169 = ms.addMenu(document.getElementById("menu_1169"));
menu1169.addItem("Graduates","/publish/zhpyen/1179/index.html","");
        menu1169.addItem("Courses","/publish/zhpyen/1180/index.html","");
        var menu1171 = ms.addMenu(document.getElementById("menu_1171"));
menu1171.addItem("PhD","/publish/zhpyen/1183/index.html","");
        menu1171.addItem("Postdoc","/publish/zhpyen/1184/index.html","");
        menu1171.addItem("Faculty","/publish/zhpyen/1185/index.html","");
        mtDropDown.renderAll();
}
} 
</SCRIPT>
<div id="nav">
    <ul>
      <li><a href="/publish/zhpyen/index.html" title="Home">Home</a></li>
      <li id="menu_1167"><a href="/publish/zhpyen/1167/index.html" title="About">About</a></li>
      <li id="menu_1170"><a href="/publish/zhpyen/1181/index.html" title="Research">Research</a></li>
      <li id="menu_1169"><a href="/publish/zhpyen/1179/index.html" title="Teaching">Teaching</a></li>
      <li id="menu_1168"><a href="/publish/zhpyen/1176/index.html" title="People">People</a></li>

      <li id="menu_1171"><a href="/publish/zhpyen/1183/index.html" title="Opportunities">Opportunities</a></li>
      <li id="menu_1172"><a href="/publish/zhpyen/1172/index.html" title="Contact us">Contact us</a></li>
    </ul>
   </div>
<script language="JavaScript">initMenuData();</script>  <!--nav结束处-->
  <!--s_content开始处-->
  <div id="s_content">
   <!--detail_2开始处-->
   <div class="detail_2">
            
            <!-- c_top start -->

<div class="c_top">
                <ul class="nav_01">
                    <li>&nbsp;<A href='/publish/zhpyen/index.html' class='navigation_style'>Home</a>&nbsp;&nbsp;&#8250;&nbsp;&nbsp;<A href='/publish/zhpyen/1181/index.html' class='navigation_style'>Research</a>&nbsp;&nbsp;&#8250;&nbsp;&nbsp;<A href='/publish/zhpyen/1181/index.html' class='navigation_style'>Areas</a>&nbsp;&nbsp;&#8250;&nbsp;&nbsp;Content</li>
                </ul>
            </div>            <!-- c_bottom end -->
            
            <!-- start -->
            <div class="box_detail">
              <h2><span>Protein Structure and Protein Folding</span></h2>
              <p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">Recombinant DNA techniques have provided tools for the rapid determination of DNA sequences and, by inference, the amino acid sequences of proteins from structural genes. The proteins we observe in nature have evolved, through selective pressure, to perform specific functions. The function properties of proteins depends upon their three-dimensional structures[4].</span></span></span></p><p style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><img alt="" src="/publish/zhpyen/1181/20101212175322899680164/protein1.jpg" width="255" height="197"/></span></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">&nbsp;&nbsp;&nbsp; &nbsp; To understand the biological function of proteins we would therefore like to be able to deduce or predict the three-dimensional structure from the amino acid sequences. However, in spite of considerable efforts over the past 30 years, this folding problems is still unsolved and remains one of the most basic intellectual challenges in molecular biology[4].</span></span></span></p><p style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><img alt="" src="/publish/zhpyen/1181/20101212175322899680164/protein2(2).jpg" width="273" height="200"/>&nbsp;</p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;According to the experiments first performed by Christian Anfinsen in the 1950s, protein unfolding and refolding are thermodynamically reversible process. By changing the environment conditions, the protein unfolds and loses its activity, but once the environment is returned to the physiological condition the protein folds up spontaneously to its native structure and regains its activity. Thus, protein folding is apparently thermodynamically determined[7]. Anfinsen referred this as the &ldquo;thermodynamic hypothesis&rdquo;. This hypothesis states that the three-dimensional structure of a native protein in its normal physiological milieu is the one in which the Gibbs free energy of the whole system is lowest; that is that the native conformation is determined by the totality of interatomic interactions and hence by the amino acid sequence, in a given environment[2].</span></span></span></p><p style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><img alt="" src="/publish/zhpyen/1181/20101212175322899680164/protein3.jpg" width="222" height="227"/></span></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; From the thermodynamic hypothesis, the amino acid sequence appears to contain all the necessary information to make up the native 3D structure. Basic on this principle, most of the recent methods to predict the native 3D structure of a protein from its amino acid sequence are semi-empirical. Statistical methods are widely used in the prediction. The general idea for such prediction includes two steps:</span></span></span><span style="FONT-FAMILY: &#39;Times New Roman&#39;,&#39;serif&#39;; mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><span style="FONT-SIZE: 16px"><?xml:namespace prefix="o"><o:p></o:p></?xml:namespace></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;1. Predict the secondary structures (or structure of some segments) by threading or alignment;</span></span></span><span style="FONT-FAMILY: &#39;Times New Roman&#39;,&#39;serif&#39;; mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><span style="FONT-SIZE: 16px"><o:p></o:p></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;2. Assemble the segments to minimize the energy to find out the 3D structure. </span></span></span><span style="FONT-FAMILY: &#39;Times New Roman&#39;,&#39;serif&#39;; mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><span style="FONT-SIZE: 16px"><o:p></o:p></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">However, being not know the principle that how a protein folds to its &ldquo;unique&rdquo; native state, such methods are success only for specific proteins. </span></span></span><span style="FONT-FAMILY: &#39;Times New Roman&#39;,&#39;serif&#39;; mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><span style="FONT-SIZE: 16px"><o:p></o:p></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">&nbsp;&nbsp;&nbsp; &nbsp; In addition to the semi-empirical method, some people try to predict the native structure by searching the structure with lowest free energy. However, this can not be done yet. Because of computational limitations, it is not possible to generate an ensemble of structures that adequately represent the equilibrium ensemble for even the smallest globular proteins[9]. Moreover, there is still debate over whether the native folded state of the protein represents a thermodynamically dominant state or a kinetically trapped state[8].</span></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;In [8], the authors points out that one central question is whether this conformation (the native conformation) is under thermodynamic or kinetic control; i.e., Whether the native protein conformation corresponding to the most stable (thermodynamic control) or to the kinetically most accessible (kinetic control) conformation. There has been no proof of either of the two situations. The original experiments of Anfinsen seem to support the thermodynamic hypothesis. Recent experiments, however, have suggest that there may be exceptions, especially for larger and more complex proteins[3]. It seems likely that kinetic control is more prevalent in complex cellular processes.</span></span></span></p><p style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><img alt="" src="/publish/zhpyen/1181/20101212175322899680164/protein4.jpg" width="458" height="200"/></span></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The issue for this debate is to calify the condition under which a protein would folds follow the thermodynamic control.</span></span></span><span style="FONT-FAMILY: &#39;Times New Roman&#39;,&#39;serif&#39;; mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><span style="FONT-SIZE: 16px"><o:p></o:p></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;For those proteins that follow thermodynamics control, prediction is possible in principle, by attempting to find the most stable state.&nbsp;However, for those proteins under kinetic control, it is impossible to predict the &ldquo;native conformation&rdquo; unless the mechanisms of folding are known.</span></span></span><span style="FONT-FAMILY: &#39;Times New Roman&#39;,&#39;serif&#39;; mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><span style="FONT-SIZE: 16px"><o:p></o:p></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Instead of attempting to resolve the problem of protein folding just by devise more powerful mathematical and computational methods, we propose to adopt a different and more basic approach. We wish first to examine the problem of protein folding by understanding the mechanisms of protein folding. This breaks up the complex problem of prediction into several simpler problems, each with its own characteristics.</span></span></span><span style="FONT-FAMILY: &#39;Times New Roman&#39;,&#39;serif&#39;; mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><span style="FONT-SIZE: 16px"><o:p></o:p></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; For the mechanisms of protein folding at different stages are different in general nature, and may require different kinds of mathematical methods for its prediction.</span></span></span></p><p style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><img alt="" src="/publish/zhpyen/1181/20101212175322899680164/protein5.jpg" width="245" height="206"/></span></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Through the mechanisms of protein folding, we will able to discover the general principles that govern the folding process of protein as a long chain molecule; the main feature that distinguish the native structure from the denature structure; the guide line for predicting the native structure of a protein from the amino acid sequence.</span></span></span><span style="FONT-FAMILY: &#39;Times New Roman&#39;,&#39;serif&#39;; mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><span style="FONT-SIZE: 16px"><o:p></o:p></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Protein folding is a new area for applied mathematics; long term study is required understand the mechanisms for the folding process of a protein. The main questions are outlined as following:</span></span></span><span style="FONT-FAMILY: &#39;Times New Roman&#39;,&#39;serif&#39;; mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><span style="FONT-SIZE: 16px"><o:p></o:p></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 1. The word &ldquo;native structure&rdquo; is used biologically to refer the 3D structure of a protein at physiological condition. However, the exact definition of the &ldquo;native structure&rdquo; is not known. Mathematically, we had to clarify that the &ldquo;native structure&rdquo; is not a single unique point in the phase space, but a ensemble of the conformations that share the same key features to enable the biological function.</span></span></span><span style="FONT-FAMILY: &#39;Times New Roman&#39;,&#39;serif&#39;; mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><span style="FONT-SIZE: 16px"><o:p></o:p></span></span> <span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">Therefore, it remain a difficult question how to describe the &ldquo;key features&rdquo; by the macroscopic properties of the stricture of a protein. This question seeks to give the definition of &ldquo;native structure&rdquo;. Obviously, to prediction the &ldquo;native structure&rdquo; of a protein, one need to define what is the &ldquo;native structure&rdquo;.</span></span></span><span style="FONT-FAMILY: &#39;Times New Roman&#39;,&#39;serif&#39;; mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><span style="FONT-SIZE: 16px"><o:p></o:p></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 2. Protein folding is a specific kinetic process of polymer molecules. Thus, the general principles and methods for polymer dynamics[5] is applicable to investigate the dynamical properties of a protein as a long chain molecule. Nevertheless, despite the simple models (Rouse model and Zimm model) for the study of polymer dynamics, it is still a difficult challenge to derive the formulation for thedynamics of a polymer that away from the equilibrium state.</span></span></span></p><p style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><img alt="" src="/publish/zhpyen/1181/20101212175322899680164/protein6.jpg" width="242" height="200"/></span></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 3. In addition to the characteristic of long chain molecule, the protein possesses special feature that there are local regular structures (the secondly structures elements). The presence of regular structure elements have significant effects for the dynamical process of protein folding. To clarify the interaction between the regular structure and irregular structure is essential to understand the mechanism of protein folding.</span></span></span><span style="FONT-FAMILY: &#39;Times New Roman&#39;,&#39;serif&#39;; mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><span style="FONT-SIZE: 16px"><o:p></o:p></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 4. Protein folding is a process of several different steps:</span></span></span><span style="FONT-FAMILY: &#39;Times New Roman&#39;,&#39;serif&#39;; mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><span style="FONT-SIZE: 16px"><o:p></o:p></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><strong><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">Unfolded state </span><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt; mso-fareast-font-family: CMSY10" lang="EN-US">&amp;rarr; </span><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">Molten globular state </span><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt; mso-fareast-font-family: CMSY10" lang="EN-US">&amp;rarr; </span><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">Folded state.</span></span></span></strong><span style="FONT-SIZE: 16px"><span style="FONT-FAMILY: &#39;Times New Roman&#39;,&#39;serif&#39;; mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><o:p></o:p></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; The processes at different states are dominated by different interactions and should be modelled by different mathematical models.</span></span></span></p><p style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><img alt="" src="/publish/zhpyen/1181/20101212175322899680164/protein7.jpg" width="280" height="250"/></span></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">&nbsp;&nbsp;&nbsp; &nbsp; 5. Protein folding is a complex process with many interactions between the molecules involve. However, the basic mechanisms can be simple from the view point of statistics and do not depends on the detail of the interactions. Thus, in studying the mechanisms, we should focus on the basic principle that do not depends on the detail interaction.</span></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><span style="LINE-HEIGHT: 115%; FONT-FAMILY: &#39;Times New Roman&#39;,&#39;serif&#39;; FONT-SIZE: 12pt; mso-fareast-font-family: 宋体; mso-ansi-language: EN-US; mso-fareast-language: ZH-CN; mso-bidi-language: AR-SA" lang="EN-US">&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;</span></span></span></span><span style="FONT-FAMILY: Times New Roman"><span style="FONT-SIZE: 15px"><span><span><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><span style="LINE-HEIGHT: 115%; mso-fareast-font-family: 宋体; mso-ansi-language: EN-US; mso-fareast-language: ZH-CN; mso-bidi-language: AR-SA" lang="EN-US">All graduates with biology and chemistry background are welcome to join our research group and be part of the team!</span></span></span></span></span></span></p><p style="MARGIN: 0cm 0cm 0pt" class="MsoNormal"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><o:p><span style="FONT-SIZE: 15px">&nbsp;</span></o:p></span></span></p><p><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><strong>Bibliography</strong></span></span></p><p style="MARGIN: 0cm 0cm 0pt" class="MsoNormal"><span style="FONT-FAMILY: &#39;Times New Roman&#39;,&#39;serif&#39;; mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><span style="FONT-SIZE: 16px"><o:p></o:p></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">[1] Akiyama, S., Takahashi, S., et. al., PNAS, 99(2002), 1329-1334. Uzawa, T., Akiyama, S., et. al., PNAS, 2004(101), 1171-1176. Kimura, T., Uzawa, T., et. al., PNAS, 102(2005), 2748-2753.</span></span></span><span style="FONT-FAMILY: &#39;Times New Roman&#39;,&#39;serif&#39;; mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><span style="FONT-SIZE: 16px"><o:p></o:p></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">[2] Anfinsen, C. B., Principles that govern the folding of protein chains, Science, 181(1973), 223-230.</span></span></span><span style="FONT-FAMILY: &#39;Times New Roman&#39;,&#39;serif&#39;; mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><span style="FONT-SIZE: 16px"><o:p></o:p></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">[3] Baker, D., Agard, D. A., Kinetics versus thermodynamics in protein folding, Biochemistry, 33(1994), 7505-7509.</span></span></span><span style="FONT-FAMILY: &#39;Times New Roman&#39;,&#39;serif&#39;; mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><span style="FONT-SIZE: 16px"><o:p></o:p></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">[4] Branden, C., Tooze, J., Introduction to Protein Structure, 2nd ed., Graland Publishing, 1998.</span></span></span><span style="FONT-FAMILY: &#39;Times New Roman&#39;,&#39;serif&#39;; mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><span style="FONT-SIZE: 16px"><o:p></o:p></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">[5] Doi, M., Edwards, S. F., The theroy of polymer dynamics. Clarendon Press, Oxford. 24-32, 1986.</span></span></span><span style="FONT-FAMILY: &#39;Times New Roman&#39;,&#39;serif&#39;; mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><span style="FONT-SIZE: 16px"><o:p></o:p></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">[6] Garrett, R. H., Grisham, C. M., Biochemistry, 2nd ed., Higher Education Press, Beijing, 2002.</span></span></span><span style="FONT-FAMILY: &#39;Times New Roman&#39;,&#39;serif&#39;; mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><span style="FONT-SIZE: 16px"><o:p></o:p></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">[7] Kanehisa, M., Post-Genome Informatics, Oxford University Press, 2001.</span></span></span><span style="FONT-FAMILY: &#39;Times New Roman&#39;,&#39;serif&#39;; mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><span style="FONT-SIZE: 16px"><o:p></o:p></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">[8] Lazaridis, T., Karplus, M., Thermodynamics of protein folding: a microscopic view, Biophys. Chem., 100(2003), 367-395.</span></span></span><span style="FONT-FAMILY: &#39;Times New Roman&#39;,&#39;serif&#39;; mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><span style="FONT-SIZE: 16px"><o:p></o:p></span></span></p><p style="TEXT-ALIGN: left; MARGIN: 0cm 0cm 0pt; mso-layout-grid-align: none" class="MsoNormal"><span style="FONT-SIZE: 15px"><span style="FONT-FAMILY: Times New Roman"><span style="mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US">[9] Straub, J. E., Protein folding and optimization algorithms, in Encyclopedia of computational chemistry, von Ragu&amp;acute;e Schleyer, P., ed., John Wiley &amp; Sons, Vol 3, 2184-2191.</span></span></span><span style="FONT-FAMILY: &#39;Times New Roman&#39;,&#39;serif&#39;; mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><span style="FONT-SIZE: 16px"><o:p></o:p></span></span></p><p>&nbsp;</p><p><span style="FONT-FAMILY: Times New Roman"><strong><span style="FONT-SIZE: 15px">Research Team:</span></strong>&nbsp; </span><table border="0" cellspacing="1" cellpadding="1" width="800"><tbody><tr><td style="TEXT-ALIGN: center"><p><img alt="" src="/publish/zhpyen/1181/20101212175322899680164/lin.jpg" width="100" height="123"/></p><span style="FONT-SIZE: 15px"><p><span style="FONT-FAMILY: Times New Roman"><span style="FONT-SIZE: 15px"><span><span><span><span><span style="mso-hansi-font-family: &#39;Times New Roman&#39;; mso-ascii-font-family: &#39;Times New Roman&#39;"><span><span><strong><span style="mso-hansi-font-family: &#39;Times New Roman&#39;; mso-ascii-font-family: &#39;Times New Roman&#39;">Chia-Chiao Lin</span></strong></span></span></span></span></span></span></span></span></span></p></span><span><span style="FONT-SIZE: 15px"><span style="FONT-SIZE: 11.5pt; mso-hansi-font-family: &#39;Times New Roman&#39;; mso-ascii-font-family: &#39;Times New Roman&#39;"><span lang="EN-US"><o:p><p style="MARGIN: 6pt 0cm; mso-line-height-alt: 0pt; mso-para-margin-top: .5gd; mso-para-margin-right: 0cm; mso-para-margin-bottom: .5gd; mso-para-margin-left: 0cm"><span style="FONT-SIZE: 14px"><span style="FONT-FAMILY: Times New Roman"><span><span><span><span lang="EN-US">AAAS Fellow</span></span></span></span></span></span></p></o:p></span></span></span></span></td><td style="TEXT-ALIGN: center"><p><img alt="" src="/publish/zhpyen/1181/20101212175322899680164/lei.jpg" width="100" height="123"/></p><span style="FONT-SIZE: 15px"><p><span style="FONT-FAMILY: Times New Roman"><span><span><span><span><span style="mso-hansi-font-family: &#39;Times New Roman&#39;; mso-ascii-font-family: &#39;Times New Roman&#39;"><span><span><strong><span style="mso-hansi-font-family: &#39;Times New Roman&#39;; mso-ascii-font-family: &#39;Times New Roman&#39;"><a href="http://166.111.93.130/~jzlei/">Jin-Zhi Lei</a></span></strong></span></span></span></span></span></span></span></span></p></span><span><span style="FONT-SIZE: 15px"><span style="FONT-SIZE: 11.5pt; mso-hansi-font-family: &#39;Times New Roman&#39;; mso-ascii-font-family: &#39;Times New Roman&#39;"><span lang="EN-US"><o:p><p style="MARGIN: 6pt 0cm; mso-line-height-alt: 0pt; mso-para-margin-top: .5gd; mso-para-margin-right: 0cm; mso-para-margin-bottom: .5gd; mso-para-margin-left: 0cm"><span style="FONT-FAMILY: Times New Roman"><span style="FONT-SIZE: 14px"><span><span><span><span lang="EN-US">Associate Researcher</span></span></span></span></span></span></p></o:p></span></span></span></span></td><td style="TEXT-ALIGN: center"><p><img alt="" src="/publish/zhpyen/1181/20101212175322899680164/sun.jpg" width="100" height="123"/></p><span style="FONT-SIZE: 15px"><p><span style="FONT-FAMILY: Times New Roman"><span><span><span><span><span style="mso-hansi-font-family: &#39;Times New Roman&#39;; mso-ascii-font-family: &#39;Times New Roman&#39;"><span><span><strong><span style="mso-hansi-font-family: &#39;Times New Roman&#39;; mso-ascii-font-family: &#39;Times New Roman&#39;"><a href="http://166.111.93.130/~sunwt/">Wei-Tao Sun</a></span></strong></span></span></span></span></span></span></span></span></p></span><span><span style="FONT-SIZE: 15px"><span style="FONT-SIZE: 11.5pt; mso-hansi-font-family: &#39;Times New Roman&#39;; mso-ascii-font-family: &#39;Times New Roman&#39;"><span lang="EN-US"><o:p><p style="MARGIN: 6pt 0cm; mso-line-height-alt: 0pt; mso-para-margin-top: .5gd; mso-para-margin-right: 0cm; mso-para-margin-bottom: .5gd; mso-para-margin-left: 0cm"><span style="FONT-FAMILY: Times New Roman"><span style="FONT-SIZE: 14px"><span><span><span><span lang="EN-US">Associate Researcher</span></span></span></span></span></span></p></o:p></span></span></span></span></td><td style="TEXT-ALIGN: center"><p><img alt="" src="/publish/zhpyen/1181/20101212175322899680164/hong.jpg" width="100" height="123"/></p><span style="FONT-SIZE: 15px"><p><span style="FONT-FAMILY: Times New Roman"><span><span><span><span><span style="mso-hansi-font-family: &#39;Times New Roman&#39;; mso-ascii-font-family: &#39;Times New Roman&#39;"><span><span><strong><span style="mso-hansi-font-family: &#39;Times New Roman&#39;; mso-ascii-font-family: &#39;Times New Roman&#39;">Liu Hong</span></strong></span></span></span></span></span></span></span></span></p></span><span><span style="FONT-SIZE: 15px"><span style="FONT-SIZE: 11.5pt; mso-hansi-font-family: &#39;Times New Roman&#39;; mso-ascii-font-family: &#39;Times New Roman&#39;"><span lang="EN-US"><o:p><span style="FONT-SIZE: 15px"><span><span lang="EN-US"><p style="MARGIN: 6pt 0cm; mso-line-height-alt: 0pt; mso-para-margin-top: .5gd; mso-para-margin-right: 0cm; mso-para-margin-bottom: .5gd; mso-para-margin-left: 0cm"><span style="FONT-FAMILY: Times New Roman"><span style="FONT-SIZE: 14px"><span><span><span><span lang="EN-US">Postdoc</span></span></span></span></span></span></p></span></span></span></o:p></span></span></span></span></td></tr></tbody></table></p><p><span style="FONT-FAMILY: &#39;Times New Roman&#39;,&#39;serif&#39;; mso-bidi-font-size: 10.5pt; mso-font-kerning: 0pt" lang="EN-US"><o:p></o:p></span></p>
            </div>
            <!-- end -->
                                       
       </div>
   <!--detail_2结束处--> 
  </div>
  <!--s_content结束处-->
  <!--footer开始处-->

<div id="footer">
 <p>Copyright © 2010 Tsinghua University All Rights Reserved. 地址：北京市海淀区清华园1号&nbsp;&nbsp;&nbsp;&nbsp;100084</p></div>  <!--footer结束处-->
 </div>
 <!--contaienr结束处-->
</body>
</html>
